src/hotspot/share/cds/heapShared.hpp

```diff
@@ -303,7 +295,7 @@ class HeapShared: AllStatic {
   static void copy_roots();
 
   static void resolve_classes_for_subgraphs(ArchivableStaticFieldInfo fields[],
-                                            int num, JavaThread* THREAD);
+                                            JavaThread* THREAD);
```

Comment:
Should this be TRAPS or JavaThread* current?

Reply:
It should be `JavaThread* current`. I'll try to fix that in a separate RFE: https://bugs.openjdk.org/browse/JDK-8293515